src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -13,6 +13,40 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class SourcePropertySymbol : SourcePropertySymbolBase
     {
+        internal bool ContainsFieldKeyword
```

Comment:
>ContainsFieldKeyword [](http://example.com/codeflow?start=22&length=20)

Consider moving this property below all "Create" methods and constructors. Also, it looks like the property is used only in one place. Unless it takes advantage of some cache, it might be better to just inline the code there. Perhaps as a local function.